examples/quickstarts/helidon-quickstart-se/src/main/java/io/helidon/examples/quickstart/se/GreetService.java

```diff
@@ -47,82 +48,101 @@
 
 public class GreetService implements Service {
 
-    /**
-     * The config value for the key {@code greeting}.
-     */
-    private final AtomicReference<String> greeting = new AtomicReference<>();
-
-    private static final JsonBuilderFactory JSON = Json.createBuilderFactory(Collections.emptyMap());
-
-    GreetService(Config config) {
-        greeting.set(config.get("app.greeting").asString().orElse("Ciao"));
-    }
-
-    /**
-     * A service registers itself by updating the routine rules.
-     * @param rules the routing rules.
-     */
-    @Override
-    public void update(Routing.Rules rules) {
-        rules
-            .get("/", this::getDefaultMessageHandler)
-            .get("/{name}", this::getMessageHandler)
-            .put("/greeting", this::updateGreetingHandler);
-    }
-
-    /**
-     * Return a wordly greeting message.
-     * @param request the server request
-     * @param response the server response
-     */
-    private void getDefaultMessageHandler(ServerRequest request,
-                                   ServerResponse response) {
-        sendResponse(response, "World");
-    }
-
-    /**
-     * Return a greeting message using the name that was provided.
-     * @param request the server request
-     * @param response the server response
-     */
-    private void getMessageHandler(ServerRequest request,
-                            ServerResponse response) {
-        String name = request.path().param("name");
-        sendResponse(response, name);
-    }
-
-    private void sendResponse(ServerResponse response, String name) {
-        String msg = String.format("%s %s!", greeting.get(), name);
-
-        JsonObject returnObject = JSON.createObjectBuilder()
-                .add("message", msg)
-                .build();
-        response.send(returnObject);
-    }
-
-    private void updateGreetingFromJson(JsonObject jo, ServerResponse response) {
-
-        if (!jo.containsKey("greeting")) {
-            JsonObject jsonErrorObject = JSON.createObjectBuilder()
-                    .add("error", "No greeting provided")
-                    .build();
-            response.status(Http.Status.BAD_REQUEST_400)
-                    .send(jsonErrorObject);
-            return;
-        }
-
-        greeting.set(jo.getString("greeting"));
-        response.status(Http.Status.NO_CONTENT_204).send();
-    }
-
-    /**
-     * Set the greeting to use in future messages.
-     * @param request the server request
-     * @param response the server response
-     */
-    private void updateGreetingHandler(ServerRequest request,
-                                       ServerResponse response) {
-        request.content().as(JsonObject.class).thenAccept(jo -> updateGreetingFromJson(jo, response));
-    }
+		/**
```

Comment:
Please avoid refactoring the entire file when not necessary.
Also make sure to use 4 spaces indent.

Reply:
Thanks, I'll minimize the changes and use 4 space indent!